packages/ui-extensions/src/surfaces/admin/components/InternalCustomerSegmentTemplate/InternalCustomerSegmentTemplate.ts

```diff
@@ -22,7 +22,10 @@ export type CustomerSegmentTemplateIcon =
   | 'followUpEmailMajor'
   | 'confettiMajor'
   | 'walletMajor'
```

Comment:
We have a map in web so this doesn't matter but `xxxxMajor` is the old Polaris naming convention we could change our names to be more aligned.

Reply:
ok - i've created [this issue](https://github.com/shop/core-issues/issues/89884) to tackle renaming. 

This will take care of the first step,  I updated this PR to include the old and new names.